include/aspect/simulator_signals.h

```diff
@@ -297,6 +297,15 @@ namespace aspect
     boost::signals2::signal<void (const SimulatorAccess<dim> &,
                                   const unsigned int iteration_count)> post_ARKode_solve;
 
+    /**
+     * A signal that is triggered when a multicomponent equilibrium is done 
+     * solving the equilibrated temperature for multiple entropy fields.
+     * An iteration count describing how many iterations were required
+     * to solve the equilibrium.
+     */
+    boost::signals2::signal<void (const SimulatorAccess<dim> &,
+                                  const unsigned int iteration_count)> post_multicomponent_equilibrium;
```

Comment:
this signal should move into the material model instead of the simulator signals (it is not a simulator signal). We should talk about how to do that best.